.github/workflows/release-t0rn.yml

```diff
@@ -0,0 +1,262 @@
+name: t0rn Rococo Release
+
+# Trigger only on tags starting with test
+on:
+  push:
+    tags:
+      - "test[0-9]+"
+
+# TODO: uncomment after tests
+# on:
+#   push:
+#     tags:
+#       - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+
+env:
+  PARACHAIN_NAME: t0rn
+  RUST_BACKTRACE: full
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-release:
+    runs-on: ["self-hosted", "rust"]
+    steps:
+      - name: ☁ Checkout git repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          token: ${{ secrets.GH_PAT }}
+
+      - name: ⚙️ Get nightly rust toolchain with wasm target
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-06-16
```

Comment:
We could bump this to sth recent now that we are at it.

Reply:
I'm not sure about bumping anything until runtime upgrade works correctly.
@chiefbiiko what do you think?

Reply:
yea, let's leave it for now